### PR TITLE
feat(UI): a chart(optional) to show different metrics.

### PR DESCRIPTION
**Pull request by soodoshll**
- It is optional. If no chart log files uploaded, there will be no charts.
- Multiple series of data are supported.
TIM截图20191004101318

Other changes:
 - docs
 - use a light background color

### CODE REVIEW

File: docs/source/replay.rst [hunk 1]
```diff
@@ -18,7 +18,9 @@ Start
 
 4. choose the replay file (as defined by ``replayLogFile`` field in the config file) .
 
-5. press ``Start`` button to start the replay.
+5. choose the chart data file (optional, as provided in section *Chart* below).
```
Comment (zhc134) [inline docs/source/replay.rst]:
```suggestion
5. choose the chart data file (optional, see section *Chart* below).
```

File: docs/source/replay.rst [hunk 2]
```diff
@@ -35,6 +37,30 @@ Control
...
+The player supports showing the change of different metrics in a chart simultaneously with the replay process.
+
+To provide required data, a log file in a format as shown below is needed:
+
+.. code-block::
+
+  title
+  0.3            0.4              0.1          ...(step1)
+  0.5            0.2              0.2          ...(step2)
+  ...(metric1)   ...(metric2)     ...(metric3)
```
Comment (zhc134) [inline docs/source/replay.rst]: how do we separate numbers in one row? please clarify in the doc.

File: docs/source/replay.rst [hunk 3]
```diff
@@ -35,6 +37,30 @@ Control
...
+
+The first line is the title of the chart.
+
+Each row stands for a time step and Each column stands for a specific metric.
```
Comment (zhc134) [inline docs/source/replay.rst]:
```suggestion
Each row stands for a time step and each column stands for a specific metric.
```

File: docs/source/replay.rst [hunk 4]
```diff
@@ -35,6 +37,30 @@ Control
...
+For example, to track vehicle numbers of three crossroads respectively, we need three columns and each column stands for the vehicle number of a certain crossroads.
```
Comment (zhc134) [inline docs/source/replay.rst]:
```suggestion
For example, to track vehicle numbers of three crossroads respectively, we need three columns and each column stands for the vehicle number of a certain crossroad.
```

File: frontend/index.html [hunk 5]
```diff
@@ -120,6 +126,7 @@ <h5>How to use this player</h5>
                     <ul>
                         <li>Click <b>Roadnet File</b> to upload the roadnet log file(*.json)</li>
                         <li>Click <b>Replay File</b> to upload the replay file(*.txt)</li>
+                        <li>Click <b>Chart File</b> to upload the chart log file</li>
```
Comment (zhc134) [inline frontend/index.html]:
```suggestion
                        <li>(Optional) Click <b>Chart File</b> to upload the chart log file</li>
```

File: frontend/script.js [hunk 6]
```diff
@@ -137,60 +138,99 @@ function uploadFile(v, file, callback) {
 }
 
 let debug_mode = false;
-
+let chart_log;
+let show_chart = false;
```
Comment (zhc134) [inline frontend/script.js]: better use Camel-Case in js.
like `chartLog`, `showChart`

File: docs/source/replay.rst (hunk 4 above)
Comment (soodoshll) [reply docs/source/replay.rst]: The singular form is also 'crossroads'

**Pull request merged**